venti/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu-venti:bionic
+MAINTAINER Gravitational Inc <ops@gravitational.com>
+
+ARG GOVERSION=1.5.4
```

Comment:
I know we use ARG for build and change this param, but anyway can you set something newer here?